### PR TITLE
SSE-3329: Updated Project structure

### PR DESCRIPTION
**Pull request by raghu-kora**
https://govukverify.atlassian.net/browse/SSE-3329

- Create a separate Test directory with its package.json 
- create a separate test folder and categorise acceptance and accessible tests 
- Update the support directory to support the test execution
- Add relevant scripts to execute the Tests
- Update README

### CODE REVIEW

File: ui-automation-tests/support/steps/register-steps.ts [hunk 1]
```diff
@@ -39,6 +39,8 @@ Then("the error message with the text {string} must be displayed for the {} fiel
 
 async function selectOption(page: Page, option: string, value: string) {
     const radio = await page.$x(`//div[@id="${option}-options"]//input[@value="${value}"]`);
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```
Comment (DStumpGDS) [inline ui-automation-tests/support/steps/register-steps.ts:42]: It looks like Line 42 is disabling Line 43 so can't you just remove both?

File: ui-automation-tests/support/test-setup.ts [hunk 2]
```diff
@@ -1,19 +1,33 @@
...
+    private browserPage: Page | undefined;
+
     constructor(options: IWorldOptions) {
         super(options);
     }
 
     async goToPath(path: string) {
-        await this.page.goto(new URL(path, this.host).toString());
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-ignore
```
Comment (DStumpGDS) [inline ui-automation-tests/support/test-setup.ts:17]: Ditto above comment with these Lines (16-17)?

File: ui-automation-tests/support/steps/register-steps.ts (hunk 1 above)
Comment (raghu-kora) [reply ui-automation-tests/support/steps/register-steps.ts:42]: we need this for now David due to linting

File: ui-automation-tests/support/test-setup.ts (hunk 2 above)
Comment (raghu-kora) [reply ui-automation-tests/support/test-setup.ts:17]: same here

**Review by DStumpGDS — APPROVED** [review]
Comments addresses so happy to approve.

**Review by DStumpGDS — APPROVED** [review]
Commits now squashed so happy to re-approve.

**Pull request merged**